hs/KernelPerceptron/BudgetPythonNumpy.py

```diff
@@ -0,0 +1,138 @@
+#Python Implementation of the Budget Kernel Perceptron
```

Comment:
I see `Numpy` in the filename. But can the comment clarify the difference between this implementation and the `BudgetPython.py` one?

Reply:
Strike this comment :-) I only later read your excellent documentation file.

Reply:
Or perhaps just add a pointer in the various `.py` files to the documentation file.

Reply:
Can do!